SPECS/openssh/openssh.spec

```diff
@@ -1,15 +1,15 @@
 %global openssh_ver 8.5p1
```

Comment:
Should we bump the release to 5 ?
Double checking .. does not seem like we do .. still

Reply:
Nope, we just haven't built this one even once, yet, so there's no need.